src/waypoint.jsx

```diff
@@ -178,6 +207,17 @@ export default class Waypoint extends React.Component {
     clearTimeout(this.initialTimeout);
   }
 
+  _ensureChildrenIsSingleDOMElement(children) {
```

Comment:
Since this function doesn't reference `this`, it doesn't need to be on the class. Move it up to the module.